datatypes.go

```diff
@@ -1312,6 +1322,10 @@ type CredentialCreate struct {
 	// y=example, etc. A rule of '*' is equivalent to no acl at all and will explicitly
 	// permit all actions.
 	ACL []string `json:"acl,omitempty"`
+	// If supplied at credential creation, ownership will be assigned to the specified
+	// User or Bot. Only admins may specify an owner other than themselves. Defaults to
+	// the authenticated User or Bot.
+	OwnerId *string `json:"owner_id,omitempty"`
```

Comment:
This one's go name should probably be "OwnerID", and it's probably safest to fix that before tagging a release since it's a breaking change to update it after the fact :|

Reply:
Updated internally, we should be able to regen api changes. I'll give it a go..